homeassistant/components/alarm_control_panel/abode.py

```diff
@@ -54,26 +52,30 @@ def icon(self):
     @property
     def state(self):
         """Return the state of the device."""
-        status = self._device.mode
+        state = STATE_UNKNOWN
```

Comment:
Have `None` be the default. It will be translated to `STATE_UNKNOWN` by the `Entity` base class.